app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/utils/DropDownHelper.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.views.utils
+
+import javax.inject.Inject
+import play.api.i18n.Messages
+import play.twirl.api.Html
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.utils.HtmlUtil
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.paragraph_block
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.bullets
+
+class DropDownHelper @Inject() (paragraph: paragraph_block, bulletList: bullets) extends HtmlUtil {
+
+  val landingKey: String          = "landing.details"
+  val supportingEvidenceUploadKey = "supporting-evidence.upload.details"
+  val reasonAndBasisBasisKey      = "select-reason-and-basis-for-claim.basis.details"
+  val reasonAndBasisReasonKey     = "select-reason-and-basis-for-claim.reason.details"
+  val basisKey                    = "select-basis-for-claim.details"
+  val whoIsMakingTheClaimKey      = "select-who-is-making-the-claim.details"
+
+  def landingDropDownContent()(implicit messages: Messages): Html =
```

Comment:
There's a general problem with all the helpers in the `util` folder. We need to pass in the actual request to this helper methods (implicitly) so that they pick up the correct language.

Reply:
Will create a ticket and address this later since we are not going live in March with Welsh